wiki/en/Running-a-Server.md

```diff
@@ -84,11 +73,11 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
 To run a Custom Directory [read this guide](Custom-Directories)
```

Comment:
I'll say it again - I don't like the heading being "Custom Directory".  It's just "Directory".  It's what Volker does for Any Genre 1, 2 and 3 and what I do for the genre-specific Directories.  And Volker does for at least one other and I do for four others.  Mine run identically except for port number (and welcome message).

Reply:
Changed